content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -226,7 +226,7 @@ kubectl get pods --all-namespaces -o jsonpath='{range .items[*].status.initConta
 kubectl get events --sort-by=.metadata.creationTimestamp
 
 # List all warning events
-kubectl events --types=Warning
+kubectl get events --types=Warning
```

Comment:
The `kubectl events --types=Warning` is correct. Previously it was `kubectl alpha events` but from k8s v1.26 this moved from `kubectl alpha events` to `kubectl events`.

See this: https://github.com/kubernetes/kubernetes/pull/113819

Reply:
Got it @Shubham82. FWIW, `kubectl events` is not available in v1.26.1 current available in homebrew.

Reply:
I wouldn't expect this to be listed in the docs if not available in all versions of v1.26.x— is this a wrong assumption?

Reply:
I checked on kubectl version v1.26, and for me, it is working. it will be available from the v1.26 version of kubectl.

 ```
 kubectl version --short
Flag --short has been deprecated, and will be removed in the future. The --short output will become the default.
Client Version: v1.26.1
Kustomize Version: v4.5.7
Server Version: v1.26.0
```
```
$ kubectl events --help
Display events

 Prints a table of the most important information about events. You can request events for a namespace, for all
namespace, or filtered to only those pertaining to a specified resource.

Examples:
  # List recent events in the default namespace.
  kubectl events
  
  # List recent events in all namespaces.
  kubectl events --all-namespaces
  
  # List recent events for the specified pod, then wait for more events and list them as they arrive.
  kubectl events --for pod/web-pod-13je7 --watch
  
  # List recent events in given format. Supported ones, apart from default, are json and yaml.
  kubectl events -oyaml
  
  # List recent only events in given event types
  kubectl events --types=Warning,Normal

Options:
    -A, --all-namespaces=false:
	If present, list the requested object(s) across all namespaces. Namespace in current context is ignored even
	if specified with --namespace.

    --allow-missing-template-keys=true:
	If true, ignore any errors in templates when a field or map key is missing in the template. Only applies to
	golang and jsonpath output formats.

    --chunk-size=500:
	Return large lists in chunks rather than all at once. Pass 0 to disable. This flag is beta and may change in
	the future.

    --for='':
	Filter events to only those pertaining to the specified resource.

    --no-headers=false:
	When using the default output format, don't print headers.

    -o, --output='':
	Output format. One of: (json, yaml, name, go-template, go-template-file, template, templatefile, jsonpath,
	jsonpath-as-json, jsonpath-file).

    --show-managed-fields=false:
	If true, keep the managedFields when printing objects in JSON or YAML format.

    --template='':
	Template string or path to template file to use when -o=go-template, -o=go-template-file. The template format
	is golang templates [http://golang.org/pkg/text/template/#pkg-overview].

    --types=[]:
	Output only events of given types.

    -w, --watch=false:
	After listing the requested events, watch for more events.

Usage:
  kubectl events
[(-o|--output=)json|yaml|name|go-template|go-template-file|template|templatefile|jsonpath|jsonpath-as-json|jsonpath-file]
[--for TYPE/NAME] [--watch] [--event=Normal,Warning] [options]

Use "kubectl options" for a list of global command-line options (applies to all commands).

```

Reply:
which version of kubectl you are using?